database/migrations/2016_03_13_203345_create_members_table.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateMembersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('members', function (Blueprint $table) {
+            $table->increments('id');
+
+            $table->string('last_name');
+            $table->string('first_name');
+            $table->enum('gender', ['Male', 'Female']);
+
+            // Nationality and DOB are nullable because some people do not want to declare.
+            $table->date('date_of_birth')->nullable();
+            $table->string('nationality')->nullable();
+
+            $table->boolean('is_singapore_pr');
+            $table->string('nric');
+            $table->string('crsid')->unique();
```

Comment:
we should make crsid nullable. Some non university members might join cumsa @spencerthang